### ISSUE TITLE
Document the CLI options for yark refresh

### ISSUE THREAD

**Issue by gotofritz** [Owez/yark#124]
```
❯ yark -h refresh # there is no difference
❯ yark -h 
yark [options]

  YouTube archiving made simple.

Options:
  new [name] [url]         Creates new archive with name and channel url
  refresh [name] [args?]   Refreshes/downloads archive with optional config
  ...
```

Nowhere does it say what those `args` are.

**Comment by gotofritz** [comment]
By looking at the source, these are

```
--videos=N # download the N latest videos
--livestreams=N # download the N latest livestreams
--shorts=N # download the N latest shorts
--skip-metadata
--skip-download
--format # Custom yt-dlp format
```

**Comment by gotofritz** [comment]
Submitted a PR

**Comment by Owez** [comment]
Closed via closure of #125

**Comment by gotofritz** [comment]
You are right, not sure how I missed it, I must have mystyped the command 🙄
